docs/install/docker-security.md

```diff
@@ -0,0 +1,198 @@
+        hard: -1
+    volumes:
+      - odfe-data2:/usr/share/elasticsearch/data
+      - ./root-ca.pem:/usr/share/elasticsearch/config/root-ca.pem
+      - ./esnode.pem:/usr/share/elasticsearch/config/esnode.pem
+      - ./esnode-key.pem:/usr/share/elasticsearch/config/esnode-key.pem
+      - ./kirk.pem:/usr/share/elasticsearch/config/kirk.pem
+      - ./kirk-key.pem:/usr/share/elasticsearch/config/kirk-key.pem
+      - ./custom-elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
+      - ./internal_users.yml:/usr/share/elasticsearch/plugins/opendistro_security/securityconfig/internal_users.yml
+    networks:
+      - odfe-net
+  kibana:
+    image: amazon/opendistro-for-elasticsearch-kibana:0.7.0
+    container_name: odfe-kibana
+    ports:
+      - 5601:5601
+    expose:
+      - "5601"
+    environment:
+      ELASTICSEARCH_URL: https://odfe-node1:9200
+    volumes:
+      - ./custom-kibana.yml:/usr/share/kibana/config/kibana.yml
+    networks:
+      - odfe-net
+
+volumes:
+  odfe-data1:
+  odfe-data2:
+
+networks:
+  odfe-net:
+```
+
+Then make your changes to `elasticsearch.yml`. For a full list of settings, see [Security](../../security). This example adds (extremely) verbose audit logging:
+
+```yml
+opendistro_security.ssl.transport.pemcert_filepath: esnode.pem
+opendistro_security.ssl.transport.pemkey_filepath: esnode-key.pem
+opendistro_security.ssl.transport.pemtrustedcas_filepath: root-ca.pem
+opendistro_security.ssl.transport.enforce_hostname_verification: false
+opendistro_security.ssl.http.enabled: true
+opendistro_security.ssl.http.pemcert_filepath: esnode.pem
+opendistro_security.ssl.http.pemkey_filepath: esnode-key.pem
+opendistro_security.ssl.http.pemtrustedcas_filepath: root-ca.pem
+opendistro_security.allow_unsafe_democertificates: true
+opendistro_security.allow_default_init_securityindex: true
+opendistro_security.authcz.admin_dn:
+  - CN=kirk,OU=client,O=client,L=test, C=de
+
+opendistro_security.audit.type: internal_elasticsearch
+opendistro_security.enable_snapshot_restore_privilege: true
+opendistro_security.check_snapshot_restore_write_privileges: true
+opendistro_security.restapi.roles_enabled: ["all_access", "security_rest_api_access"]
+cluster.routing.allocation.disk.threshold_enabled: false
+opendistro_security.audit.config.disabled_rest_categories: NONE
+opendistro_security.audit.config.disabled_transport_categories: NONE
+```
+
+To start the cluster, run `docker-compose up`.
+
+If you encounter any `File /usr/share/elasticsearch/config/elasticsearch.yml has insecure file permissions (should be 0600)` messages, you can use `chmod` on your local machine to set file permissions before running `docker-compose up`. Docker Compose passes files to the container as-is.
+{: .note }
+
+
+## Change passwords for read-only users
+
+After the cluster starts, change the passwords for the read-only user accounts (`admin` and `kibanaserver`). Run `docker ps` to find the `odfe-node1` container ID. Then run:
```

Comment:
We might want to add a sentence or link to what a read-only user account it.

Reply:
Addressed in latest.